static/js/controllers/inbox.js

```diff
@@ -12,9 +12,9 @@ var utils = require('kujua-utils'),
 
   var inboxControllers = angular.module('inboxControllers', []);
 
-  inboxControllers.controller('InboxCtrl', 
-    ['$scope', '$route', '$location', '$translate', '$animate', '$rootScope', 'Facility', 'Form', 'Settings', 'Contact', 'Language', 'ReadMessages', 'UpdateUser', 'SendMessage', 'User', 'UserDistrict', 'UserCtxService', 'Verified', 'DeleteMessage', 'UpdateFacility', 'Exports',
-    function ($scope, $route, $location, $translate, $animate, $rootScope, Facility, Form, Settings, Contact, Language, ReadMessages, UpdateUser, SendMessage, User, UserDistrict, UserCtxService, Verified, DeleteMessage, UpdateFacility, Exports) {
+  inboxControllers.controller('InboxCtrl',
+    ['$scope', '$route', '$location', '$translate', '$animate', '$rootScope', 'FacilityRaw', 'Form', 'Settings', 'Contact', 'Language', 'ReadMessages', 'UpdateUser', 'SendMessage', 'User', 'UserDistrict', 'UserCtxService', 'Verified', 'DeleteMessage', 'UpdateFacility', 'Exports',
```

Comment:
The FacilityRaw service was designed to be private (but it might work all the same). I think it'd be better to use the Facility service and with options of `{ types: [ 'clinic' ] }` which will do the filtering for you.